site/content/en/docs/tutorials/setup_minikube_in_china.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "minikube start"
+linkTitle: "Get Started!"
+weight: 1
+---
+
+This Tutorial explain how to setup minikube in china. For more detail please refer [minikube installation guide]({{<ref "/docs/start">}})
+
+<h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">1</strong></span>Installation</h2>
+
+{{% tabs %}}
+{{% tab "Linux" %}}
+
+### Binary download
+
+```shell
+ curl -Lo minikube https://github.com/kubernetes/minikube/releases/download/v1.12.2/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
```

Comment:
this installation document is same as normal users.
the parts that are same to normal users, Give a link to the user to follow

just tell them install minikube this way...

and only document the part that is different for normal users